### DISCUSSION TITLE
extrude in either +z or -z

### DISCUSSION THREAD

**Discussion by kmahajan-cadfem** [ansys/pyansys-geometry#1040] [Q&A] [answered]
by default the `extrude_sketch` will be done in plane normal direction and that should be default.
is there any way that we can create 2 sketches on same plane, one to extrude in one direction(say +z) and another in another direction (say -z).

I am aware that I can just change the plane or create new sketch but i would like to keep sketches on same plane.

**Comment by RobPasMue** [comment, marked as answer]
I'm sure you tried (I haven't though) but... have you tried passing in a "negative" distance?

**Reply by RobPasMue** [reply]
If that works - we can easily implement it on the client side.

**Reply by RobPasMue** [reply]
If it doesn't, it can be easily implemented still =)

**Reply by kmahajan-cadfem** [reply]
That works out of the box, I was still treating it as a distance so could not think that I could add the negative values

**Reply by RobPasMue** [reply]
Awesome! In any case - I am working on something to make it easier... let me ping you in the PR once I have it.
